pinot-connectors/pinot-flink-connector/src/main/java/org/apache/pinot/connector/flink/sink/PinotSinkFunction.java

```diff
@@ -60,13 +63,17 @@ public class PinotSinkFunction<T> extends RichSinkFunction<T> implements Checkpo
   private TableConfig _tableConfig;
   private Schema _schema;
 
+  private String _segmentNamePrefix;
+  @Nullable private Long _overriddenSegmentCreationTimeMs;
```

Comment:
let's keep the class variable name as `segmentCreationTimeMs` here as having `overridden` may be confusing given we are not overriding any other value in this class with this variable.

Reply:
The idea is to have either the job start time in segment name or if not provided then current time when segment is flused. Can rename it if this is the naming convention followed for similar scenarios.

Reply:
Hmm the variable name makes sense in `FlinkSegmentWriter` but in this class it looks like out of context imo.

Reply:
Let me rename it here with a code comment.